src/README.md

```diff
@@ -24,6 +29,33 @@ You should see the following content if it runs correctly.
 Some api for developers has been added, check them out in route.py.
 The HTML templates are to represent the view of administrators.
 
+### Deploying on Heroku
+We will teach you how to use Heroku Cli to deploy the website
+First, create a heroku app and then connect it.
+```
+heroku create
+heroku git:remote -a "Your app name"
+```
+
+If you want to create a postgresql, the following command can help you.
+```
+heroku addons:create heroku-postgresql:hobby-dev
+```
+
+You should create a .ENV file includeing the following content.
+```
+DATABASE_URL="Your database url" or you will use sqlite as your database
```

Comment:
It would be better to give an example about the DATABASE_URL. The `heroku addons:create heroku-postgresql:hobby-dev` would yield `Created postgresql-slippery-51678 as DATABASE_URL` and I'm not sure if it is exactly the DATABASE_URL here.